README.md

```
@@ -330,4 +331,66 @@ Content-Type: application/json
     "themeName" : string
     "status" : string
 }
+```
+
+### 관리자 예약대기 목록 조회
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)